browser/mapping.go

```diff
@@ -439,7 +439,9 @@ func mapPage(vu moduleVU, p api.Page) mapping {
 				return nil, err //nolint:wrapcheck
 			})
 		},
-		"close":                   p.Close,
+		"close": func(opts goja.Value) error {
+			return p.Close(opts) //nolint:wrapcheck
+		},
```

Comment:
Can we simply do:

```suggestion
		"close": p.Close(opts),
```

Reply:
I tried this, but then realised that the opts haven't been defined. So reverting this to what it was.